src/mono/mono/mini/mini-amd64.c

```diff
@@ -6798,6 +6798,18 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 				amd64_sse_xorps_reg_reg (code, ins->dreg, SIMD_TEMP_REG);
 				break;
 			}
+			case MONO_TYPE_I:
+			case MONO_TYPE_U:
+#if TARGET_SIZEOF_VOID_P == 8
```

Comment:
is this always true ?

Reply:
These should be added to the MONO_TYPE_I8 block above.